pkg/deploy/lattice/service_network_manager.go

```diff
@@ -53,66 +57,72 @@ type defaultServiceNetworkManager struct {
 // return errors.New(LATTICE_RETRY) when:
 //
 //	CreateServiceNetworkVpcAssociationInput returns ServiceNetworkVpcAssociationStatusFailed/ServiceNetworkVpcAssociationStatusCreateInProgress/MeshVpcAssociationStatusDeleteInProgress
-func (m *defaultServiceNetworkManager) Create(ctx context.Context, service_network *latticemodel.ServiceNetwork) (latticemodel.ServiceNetworkStatus, error) {
+func (m *defaultServiceNetworkManager) CreateOrUpdate(ctx context.Context, stackSn *latticemodel.ServiceNetwork) (latticemodel.ServiceNetworkStatus, error) {
```

Comment:
why stackSn, not just sn?

Reply:
Hope to to represent a meaning that this a the desired SN that user intend to create (refering the name `stackDeployer`)

Reply:
looks cryptic, sn or serviceNetwork was ok

Reply:
I found it have to differenate desiredSN and m.cloud.Lattice().FindServiceNetwork() retrieved SN, plan to change the name `stackSn` to `desiredSN`

Reply:
I believe difference is apparent, Create function accept model of service network and tries to create or update lattice resources. There is no ambiguity what kind of service network it is.

Reply:
Changed to `serviceNetwork`